app/touchpoint/TouchpointBackendConfig.scala

```diff
@@ -11,7 +11,8 @@ import org.joda.time.Period
 case class TouchpointBackendConfig(
```

Comment:
A lot of this class is becoming a duplicate of what's in the TouchpointBackendConfig in common, maybe a good chunk of it can be removed soon.